src/zenml/stack_stores/__init__.py

```diff
@@ -11,7 +11,10 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 #  or implied. See the License for the specific language governing
 #  permissions and limitations under the License.
-"""Stack Stores."""
+"""
+The stack store defines exactly where and how stacks are persisted across their 
```

Comment:
Not sure if on purpose, but the other two docstrings for stacks and step operators start with a `##` title

Reply:
Yeah those are when people do `zenml subocmmand explain`, but are not relevant for stack stores